cmd/clairctl/report.go

```diff
@@ -174,32 +186,55 @@ func reportAction(c *cli.Context) error {
 			//
 			// If we need the manifest, populate the manifest and jump to Again.
 			var m *claircore.Manifest
+			ct := 1
 		Again:
+			if ct > 20 {
```

Comment:
What are you thoughts on the retry logic being encapsulated in the client? Feels like the client could eventually become it's own thing and the retry logic would be nice to have by default.

Reply:
I think this is a good eventual goal, but since the client is internal right now, I'm not too worried about the occasional layering leak like this one.